tests/functional/firefox/os/test_tv.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.os.tv import TVPage
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_next_previous_buttons(base_url, selenium):
+    page = TVPage(base_url, selenium).open()
+    assert not page.is_previous_enabled
+    screens = page.screens
+    thumbnails = page.thumbnails
+    for i in range(len(screens))[:-1]:
+        assert screens[i].is_displayed
+        assert thumbnails[i].is_selected
+        page.click_next()
+    assert not page.is_next_enabled
+    for i in range(len(screens))[::-1][:-1]:
```

Comment:
Yeah, it seems `reversed` can be used but it's not as clean as I'd hoped. I think we can make this a little cleaner though by using the start, stop, step arguments for `range`. I've testing this locally:

```python
for i in range(len(screens) - 1):
    assert screens[i].is_displayed
    assert thumbnails[i].is_selected
    page.click_next()
assert not page.is_next_enabled
for i in range(len(screens) - 1, 0, -1):
    assert screens[i].is_displayed
    assert thumbnails[i].is_selected
    page.click_previous()
```